.github/workflows/_build-plugin-binaries.yml

```diff
@@ -0,0 +1,142 @@
+name: Build snforge_scarb_plugin
+
+on:
+  workflow_call:
+    inputs:
+      overridden_plugin_version:
+        required: false
+        type: string
+  workflow_dispatch:
+    inputs:
+      overridden_plugin_version:
+        required: false
+        type: string
+
+jobs:
+  check-version:
+    name: Check snforge_scarb_plugin Version
+    runs-on: ubuntu-latest
+    outputs:
+      plugin_uploaded: ${{ steps.check-version.outputs.plugin_uploaded }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Check version
+        id: check-version
+        run: |
+          set -exo pipefail
+
+          if [ -z "${{ inputs.overridden_plugin_version }}" ]; then
+            snforge_scarb_plugin_version=$(grep version crates/snforge-scarb-plugin/Scarb.toml | cut -d '"' -f 2)
```

Comment:
nit: at the moment there is only one ocurence of "version" so it's fine, but maybe we could improve this in case anything more with `.*version*` appears in the toml? I think grep-ing with regex like `^version =` could be more than enough, but am not sure. You could use jq as well.

Reply:
applicable to other places like this as well